### DISCUSSION TITLE
This video is full of scam bots

### DISCUSSION THREAD

**Discussion by FurblandChannel** [ThioJoe/YT-Spammer-Purge#885] [General]
Just decided to look through old YouTube vids on archive.org and found this, but it’s full of bots! https://youtu.be/RBshTpswT18

**Comment by FurblandChannel** [comment]
https://youtu.be/jNQXAC9IVRw Me at the zoo comments sorted by newest first is also a nightmare of bots

**Reply by Firecul** [reply]
![image](https://user-images.githubusercontent.com/34950371/178807033-05e55e12-58bc-497c-9fd4-dd6989466885.png)

I don't think we have the API quota to do anything about that one.

**Reply by FurblandChannel** [reply]
I mean we can at least try our best.

**Reply by Casondoesstuuffz** [reply]
We could team up and scan big videos together

**Reply by Firecul** [reply]
> We could team up and scan big videos together

Right now there is no way to coordinate that. At a guess the API would just return the comments in the same order to everyone and afaik we can't skip some to only scan 100,000-200,000 and miss 0-100,000 for example.
